### ISSUE TITLE
ReGaHss' exec: '/bin/triggerAlarm.tcl ReGaHss - ReGaHss: SIGTERM: ReGaHss Halting

### ISSUE THREAD

**Issue by tjschmid** [jens-maus/RaspberryMatic#1541]
### Describe the issue you are experiencing

After upgrade to version 3.61.5.20211113 the UI was not respoinding anymore. I recocnized that it will crash regulary with an ReGaHss Alert. Furthermore all central programms where not working. Direct connection were working as normal. After rolling back to 3.59.6.20211009 the whole system was working as before.

I did the upgrade twice with the same result. After the second upgrade I saved the logs you can see in my bug report. 

Finally I did a rollback to 3.59.6.20211009.

### Describe the behavior you expected

After the upgrade it should work then before.

### Steps to reproduce the issue

Just upgrading the Raspberry from 3.59.6.20211009 to 3.61.5.20211113.

### What is the version this bug report is based on?

3.61.5.20211113

### Which base platform are you running?

rpi4 (RaspberryPi4)

### Which HomeMatic/homematicIP radio module are you using?

RPI-RF-MOD

### Anything in the logs that might be useful for us?

```txt
Nov 18 10:42:01 CCU3 user.warn monit[1284]: 'ReGaHss' failed protocol test [DEFAULT] at [localhost]:31999 [TCP/IP] -- Connection timed out
Nov 18 10:42:02 CCU3 user.err rfd: Parameterset MASTER not found
Nov 18 10:42:02 CCU3 local0.warn ReGaHss: WARNING: XMLRPC 'getParamsetDescription': rpcClient.isFault() failed (url: xmlrpc_bin://127.0.0.1:32001, params: {"KEQ1070840:2","MASTER"}, result: [faultCode:-3,faultString:"Unknown paramset"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2608]
Nov 18 10:42:02 CCU3 local0.err ReGaHss: ERROR: XMLRPC 'getParamsetDescription' call failed (interface: 1007, params: {"KEQ1070840:2","MASTER"}) [GetAESMasterOperation():iseXmlRpc.cpp:2388]
Nov 18 10:42:06 CCU3 user.err rfd: Parameterset MASTER not found
Nov 18 10:42:06 CCU3 local0.warn ReGaHss: WARNING: XMLRPC 'getParamsetDescription': rpcClient.isFault() failed (url: xmlrpc_bin://127.0.0.1:32001, params: {"KEQ1070840:2","MASTER"}, result: [faultCode:-3,faultString:"Unknown paramset"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2608]
Nov 18 10:42:06 CCU3 local0.err ReGaHss: ERROR: XMLRPC 'getParamsetDescription' call failed (interface: 1007, params: {"KEQ1070840:2","MASTER"}) [GetAESMasterOperation():iseXmlRpc.cpp:2388]
Nov 18 10:42:06 CCU3 local0.warn ReGaHss: WARNING: XMLRPC 'setValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"001A5A499856BF:12","COMBINED_PARAMETER","L=30,DV=2,DU=0,RTV=0,RTU=1,C=4"}, result: [faultCode:-1,faultString:"Generic error"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2608]
Nov 18 10:42:06 CCU3 local0.err ReGaHss: ERROR: XMLRPC 'setValue' call failed (interface: 1009, params: {"001A5A499856BF:12","COMBINED_PARAMETER","L=30,DV=2,DU=0,RTV=0,RTU=1,C=4"}) [CallSetValue():iseXmlRpc.cpp:1505]
Nov 18 10:42:06 CCU3 local0.err ReGaHss: ERROR: rpc.CallSetValue failed; address = 001A5A499856BF:12 [WriteValue():iseDOMdpHSS.cpp:76]
Nov 18 10:42:12 CCU3 user.err rfd: XmlRpcClient error calling event({[methodName:"event",params:{"1007","MEQ0479338:2","ACTUAL_HUMIDITY",69.000000}],[methodName:"event",params:{"1007","MEQ0479338:2","SET_TEMPERATURE",17.000000}],[methodName:"event",params:{"1007","MEQ0479338:1","TEMPERATURE",20.000000}],[methodName:"event",params:{"1007","MEQ0479338:1","HUMIDITY",69}]}) on binary://127.0.0.1:31999/RPC2:
Nov 18 10:42:12 CCU3 user.err rfd: XmlRpc transport error
Nov 18 10:42:24 CCU3 user.err monit[1284]: 'ReGaHss' failed protocol test [DEFAULT] at [localhost]:31999 [TCP/IP] -- Connection timed out
Nov 18 10:42:24 CCU3 user.info monit[1284]: 'ReGaHss' trying to restart
Nov 18 10:42:24 CCU3 user.info monit[1284]: 'ReGaHss' restart: '/etc/init.d/S70ReGaHss restart'
Nov 18 10:42:26 CCU3 local0.warn ReGaHss: WARNING: XMLRPC 'setValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"001A5A49985695:12","COMBINED_PARAMETER","L=100,DV=2,DU=0,RTV=0,RTU=1,C=4"}, result: [faultCode:-1,faultString:"Generic error"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2608]
Nov 18 10:42:26 CCU3 local0.err ReGaHss: ERROR: XMLRPC 'setValue' call failed (interface: 1009, params: {"001A5A49985695:12","COMBINED_PARAMETER","L=100,DV=2,DU=0,RTV=0,RTU=1,C=4"}) [CallSetValue():iseXmlRpc.cpp:1505]
Nov 18 10:42:26 CCU3 local0.err ReGaHss: ERROR: rpc.CallSetValue failed; address = 001A5A49985695:12 [WriteValue():iseDOMdpHSS.cpp:76]
Nov 18 10:42:40 CCU3 user.info ReGaHss: SIGTERM: ReGaHss Halting
Nov 18 10:42:45 CCU3 daemon.info chronyd[689]: Selected source 85.220.190.246 (3.de.pool.ntp.org)
Nov 18 10:42:46 CCU3 user.err rfd: XmlRpcClient error calling event({[methodName:"event",params:{"1007","LEQ0596760:2","ACTUAL_TEMPERATURE",22.600000}]}) on binary://127.0.0.1:31999/RPC2:
Nov 18 10:42:46 CCU3 user.err rfd: XmlRpc transport error
Nov 18 10:42:46 CCU3 user.err rfd: XmlRpcClient error calling event({[methodName:"event",params:{"1007","LEQ0596760:2","ACTUAL_HUMIDITY",58.000000}]}) on binary://127.0.0.1:31999/RPC2:
Nov 18 10:42:46 CCU3 user.err rfd: XmlRpc transport error
Nov 18 10:42:46 CCU3 user.err rfd: XmlRpcClient error calling event({[methodName:"event",params:{"1007","LEQ0596760:2","SET_TEMPERATURE",20.000000}]}) on binary://127.0.0.1:31999/RPC2:
Nov 18 10:42:46 CCU3 user.err rfd: XmlRpc transport error
Nov 18 10:42:48 CCU3 user.warn monit[1284]: 'ReGaHss' failed protocol test [HTTP] at [localhost]:8183 [TCP/IP] -- Connection refused
Nov 18 10:43:01 CCU3 local0.err ReGaHss: ERROR: Error compare close tag =/obj with endtag =dom [LoadObject():iseDOMpersist.cpp:515]
Nov 18 10:43:01 CCU3 local0.err ReGaHss: ERROR: failed load OM, file= /etc/config/homematic.regadom [LoadFromFile():iseDOMpersist.cpp:68]
Nov 18 10:43:01 CCU3 local0.err ReGaHss: ERROR: LoadFromFile failed. [LoadOM():iseDOM.cpp:2577]
Nov 18 10:43:03 CCU3 user.err rfd: XmlRpcClient error calling event({[methodName:"event",params:{"1007","LEQ0929268:2","BOOT",true}]}) on binary://127.0.0.1:31999/RPC2:
Nov 18 10:43:03 CCU3 user.err rfd: XmlRpc transport error
Nov 18 10:43:03 CCU3 user.err rfd: XmlRpcClient error calling event({[methodName:"event",params:{"1007","LEQ0929268:2","ENERGY_COUNTER",5718.100000}],[methodName:"event",params:{"1007","LEQ0929268:2","POWER",1.610000}],[methodName:"event",params:{"1007","LEQ0929268:2","CURRENT",16.000000}],[methodName:"event",params:{"1007","LEQ0929268:2","VOLTAGE",232.600000}],[methodName:"event",params:{"1007","LEQ0929268:2","FREQUENCY",50.010000}]}) on binary://127.0.0.1:31999/RPC2:
Nov 18 10:43:03 CCU3 user.err rfd: XmlRpc transport error
Nov 18 10:43:05 CCU3 user.err monit[1284]: 'ReGaHss' service restarted 1 times within 1 cycles(s) - exec
Nov 18 10:43:05 CCU3 user.info monit[1284]: 'ReGaHss' exec: '/bin/triggerAlarm.tcl ReGaHss restarted WatchDog: regahss-restart true'
Nov 18 10:43:05 CCU3 user.info monit[1284]: 'ReGaHss' process is running after previous restart timeout (manually recovered?)
Nov 18 10:43:06 CCU3 user.err rfd: XmlRpcClient error calling event({[methodName:"event",params:{"1007","LEQ0596760:1","TEMPERATURE",22.600000}]}) on binary://127.0.0.1:31999/RPC2:
Nov 18 10:43:06 CCU3 user.err rfd: XmlRpc transport error
Nov 18 10:43:23 CCU3 user.err monit[1284]: 'ReGaHss' failed protocol test [DEFAULT] at [localhost]:31999 [TCP/IP] -- Connection refused
Nov 18 10:43:23 CCU3 user.info monit[1284]: 'ReGaHss' trying to restart
Nov 18 10:43:23 CCU3 user.info monit[1284]: 'ReGaHss' restart: '/etc/init.d/S70ReGaHss restart'
Nov 18 10:43:25 CCU3 local0.warn ReGaHss: WARNING: XMLRPC 'getValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"001A5A49985695:8","LEVEL"}, result: [faultCode:-5,faultString:"Unknown Parameter value for value key: LEVEL"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2608]
Nov 18 10:43:25 CCU3 local0.err ReGaHss: ERROR: XMLRPC 'getValue' call failed (interface: 1009, params: {"001A5A49985695:8","LEVEL"}) [CallGetValue():iseXmlRpc.cpp:1435]
Nov 18 10:43:25 CCU3 local0.err ReGaHss: ERROR: CallGetValue failed; sVal = 0.000000 [ReadValue():iseDOMdpHSS.cpp:124]
Nov 18 10:43:25 CCU3 local0.warn ReGaHss: WARNING: XMLRPC 'getValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"001A5A49985695:8","COLOR"}, result: [faultCode:-5,faultString:"Unknown Parameter value for value key: COLOR"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2608]
Nov 18 10:43:25 CCU3 local0.err ReGaHss: ERROR: XMLRPC 'getValue' call failed (interface: 1009, params: {"001A5A49985695:8","COLOR"}) [CallGetValue():iseXmlRpc.cpp:1435]
Nov 18 10:43:25 CCU3 local0.err ReGaHss: ERROR: CallGetValue failed; sVal = 0 [ReadValue():iseDOMdpHSS.cpp:124]
Nov 18 10:43:26 CCU3 local0.warn ReGaHss: WARNING: XMLRPC 'getValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"00165A4992426E:6","ACTIVITY_STATE"}, result: [faultCode:-5,faultString:"Unknown Parameter value for value key: ACTIVITY_STATE"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2608]
Nov 18 10:43:26 CCU3 local0.err ReGaHss: ERROR: XMLRPC 'getValue' call failed (interface: 1009, params: {"00165A4992426E:6","ACTIVITY_STATE"}) [CallGetValue():iseXmlRpc.cpp:1435]
Nov 18 10:43:26 CCU3 local0.err ReGaHss: ERROR: CallGetValue failed; sVal = 0 [ReadValue():iseDOMdpHSS.cpp:124]
Nov 18 10:43:26 CCU3 local0.warn ReGaHss: WARNING: XMLRPC 'getValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"00165A4992426E:6","LEVEL"}, result: [faultCode:-5,faultString:"Unknown Parameter value for value key: LEVEL"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2608]
Nov 18 10:43:26 CCU3 local0.err ReGaHss: ERROR: XMLRPC 'getValue' call failed (interface: 1009, params: {"00165A4992426E:6","LEVEL"}) [CallGetValue():iseXmlRpc.cpp:1435]
Nov 18 10:43:26 CCU3 local0.err ReGaHss: ERROR: CallGetValue failed; sVal = 0.000000 [ReadValue():iseDOMdpHSS.cpp:124]
Nov 18 10:43:26 CCU3 local0.warn ReGaHss: WARNING: XMLRPC 'getValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"00165A4992426E:6","LEVEL_2"}, result: [faultCode:-5,faultString:"Unknown Parameter value for value key: LEVEL_2"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2608]
Nov 18 10:43:26 CCU3 local0.err ReGaHss: ERROR: XMLRPC 'getValue' call failed (interface: 1009, params: {"00165A4992426E:6","LEVEL_2"}) [CallGetValue():iseXmlRpc.cpp:1435]
Nov 18 10:43:26 CCU3 local0.err ReGaHss: ERROR: CallGetValue failed; sVal = 0.000000 [ReadValue():iseDOMdpHSS.cpp:124]
Nov 18 10:43:26 CCU3 local0.warn ReGaHss: WARNING: XMLRPC 'getValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"00165A4992426E:6","LEVEL_2_STATUS"}, result: [faultCode:-5,faultString:"Unknown Parameter value for value key: LEVEL_2_STATUS"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2608]
Nov 18 10:43:26 CCU3 local0.err ReGaHss: ERROR: XMLRPC 'getValue' call failed (interface: 1009, params: {"00165A4992426E:6","LEVEL_2_STATUS"}) [CallGetValue():iseXmlRpc.cpp:1435]
Nov 18 10:43:26 CCU3 local0.err ReGaHss: ERROR: CallGetValue failed; sVal = 0 [ReadValue():iseDOMdpHSS.cpp:124]
Nov 18 10:43:26 CCU3 local0.warn ReGaHss: WARNING: XMLRPC 'getValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"00165A4992426E:6","LEVEL_STATUS"}, result: [faultCode:-5,faultString:"Unknown Parameter value for value key: LEVEL_STATUS"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2608]
Nov 18 10:43:26 CCU3 local0.err ReGaHss: ERROR: XMLRPC 'getValue' call failed (interface: 1009, params: {"00165A4992426E:6","LEVEL_STATUS"}) [CallGetValue():iseXmlRpc.cpp:1435]
Nov 18 10:43:26 CCU3 local0.err ReGaHss: ERROR: CallGetValue failed; sVal = 0 [ReadValue():iseDOMdpHSS.cpp:124]
Nov 18 10:43:27 CCU3 local0.warn ReGaHss: WARNING: XMLRPC 'getValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"00165A4992426E:6","PROCESS"}, result: [faultCode:-5,faultString:"Unknown Parameter value for value key: PROCESS"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2608]
Nov 18 10:43:27 CCU3 local0.err ReGaHss: ERROR: XMLRPC 'getValue' call failed (interface: 1009, params: {"00165A4992426E:6","PROCESS"}) [CallGetValue():iseXmlRpc.cpp:1435]
Nov 18 10:43:27 CCU3 local0.err ReGaHss: ERROR: CallGetValue failed; sVal = 0 [ReadValue():iseDOMdpHSS.cpp:124]
Nov 18 10:43:27 CCU3 local0.warn ReGaHss: WARNING: XMLRPC 'getValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"00165A49924209:14","LEVEL"}, result: [faultCode:-5,faultString:"Unknown Parameter value for value key: LEVEL"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2608]
Nov 18 10:43:27 CCU3 local0.err ReGaHss: ERROR: XMLRPC 'getValue' call failed (interface: 1009, params: {"00165A49924209:14","LEVEL"}) [CallGetValue():iseXmlRpc.cpp:1435]
Nov 18 10:43:27 CCU3 local0.err ReGaHss: ERROR: CallGetValue failed; sVal = 0.000000 [ReadValue():iseDOMdpHSS.cpp:124]
Nov 18 10:43:27 CCU3 local0.warn ReGaHss: WARNING: XMLRPC 'getValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"00165A4992426E:6","SECTION"}, result: [faultCode:-5,faultString:"Unknown Parameter value for value key: SECTION"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2608]
Nov 18 10:43:27 CCU3 local0.err ReGaHss: ERROR: XMLRPC 'getValue' call failed (interface: 1009, params: {"00165A4992426E:6","SECTION"}) [CallGetValue():iseXmlRpc.cpp:1435]
Nov 18 10:43:27 CCU3 local0.err ReGaHss: ERROR: CallGetValue failed; sVal = 0 [ReadValue():iseDOMdpHSS.cpp:124]
Nov 18 10:43:27 CCU3 local0.warn ReGaHss: WARNING: XMLRPC 'getValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"00165A49924209:2","LEVEL"}, result: [faultCode:-5,faultString:"Unknown Parameter value for value key: LEVEL"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2608]
Nov 18 10:43:27 CCU3 local0.err ReGaHss: ERROR: XMLRPC 'getValue' call failed (interface: 1009, params: {"00165A49924209:2","LEVEL"}) [CallGetValue():iseXmlRpc.cpp:1435]
Nov 18 10:43:27 CCU3 local0.err ReGaHss: ERROR: CallGetValue failed; sVal = 0.000000 [ReadValue():iseDOMdpHSS.cpp:124]
Nov 18 10:43:28 CCU3 local0.warn ReGaHss: WARNING: XMLRPC 'getValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"00165A4992426E:6","SECTION_STATUS"}, result: [faultCode:-5,faultString:"Unknown Parameter value for value key: SECTION_STATUS"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2608]
Nov 18 10:43:28 CCU3 local0.err ReGaHss: ERROR: XMLRPC 'getValue' call failed (interface: 1009, params: {"00165A4992426E:6","SECTION_STATUS"}) [CallGetValue():iseXmlRpc.cpp:1435]
Nov 18 10:43:28 CCU3 local0.err ReGaHss: ERROR: CallGetValue failed; sVal = 0 [ReadValue():iseDOMdpHSS.cpp:124]
Nov 18 10:43:28 CCU3 local0.warn ReGaHss: WARNING: XMLRPC 'getValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"00205BE9A06424:3","MOTION"}, result: [faultCode:-5,faultString:"Unknown Parameter value for value key: MOTION"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2608]
Nov 18 10:43:28 CCU3 local0.err ReGaHss: ERROR: XMLRPC 'getValue' call failed (interface: 1009, params: {"00205BE9A06424:3","MOTION"}) [CallGetValue():iseXmlRpc.cpp:1435]
Nov 18 10:43:28 CCU3 local0.err ReGaHss: ERROR: CallGetValue failed; sVal = 0 [ReadValue():iseDOMdpHSS.cpp:124]
Nov 18 10:43:28 CCU3 local0.warn ReGaHss: WARNING: XMLRPC 'getValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"00169BE9A0F063:10","LEVEL"}, result: [faultCode:-5,faultString:"Unknown Parameter value for value key: LEVEL"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2608]
Nov 18 10:43:28 CCU3 local0.err ReGaHss: ERROR: XMLRPC 'getValue' call failed (interface: 1009, params: {"00169BE9A0F063:10","LEVEL"}) [CallGetValue():iseXmlRpc.cpp:1435]
Nov 18 10:43:28 CCU3 local0.err ReGaHss: ERROR: CallGetValue failed; sVal = 0.000000 [ReadValue():iseDOMdpHSS.cpp:124]
Nov 18 10:43:28 CCU3 local0.warn ReGaHss: WARNING: XMLRPC 'getValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"00205BE9A06424:3","ILLUMINATION"}, result: [faultCode:-5,faultString:"Unknown Parameter value for value key: ILLUMINATION"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2608]
Nov 18 10:43:28 CCU3 local0.err ReGaHss: ERROR: XMLRPC 'getValue' call failed (interface: 1009, params: {"00205BE9A06424:3","ILLUMINATION"}) [CallGetValue():iseXmlRpc.cpp:1435]
Nov 18 10:43:28 CCU3 local0.err ReGaHss: ERROR: CallGetValue failed; sVal = 0.000000 [ReadValue():iseDOMdpHSS.cpp:124]
Nov 18 10:43:28 CCU3 local0.warn ReGaHss: WARNING: XMLRPC 'getValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"00161A499CBC31:2","STATE"}, result: [faultCode:-5,faultString:"Unknown Parameter value for value key: STATE"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2608]
Nov 18 10:43:28 CCU3 local0.err ReGaHss: ERROR: XMLRPC 'getValue' call failed (interface: 1009, params: {"00161A499CBC31:2","STATE"}) [CallGetValue():iseXmlRpc.cpp:1435]
Nov 18 10:43:28 CCU3 local0.err ReGaHss: ERROR: CallGetValue failed; sVal = 0 [ReadValue():iseDOMdpHSS.cpp:124]
Nov 18 10:43:29 CCU3 user.err rfd: HSSParameter::SetValue() true Put failed
Nov 18 10:43:29 CCU3 local0.warn ReGaHss: WARNING: XMLRPC 'setValue': rpcClient.isFault() failed (url: xmlrpc_bin://127.0.0.1:32001, params: {"KEQ1070840:2","STATE",true}, result: [faultCode:-1,faultString:"Failure"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2608]
Nov 18 10:43:29 CCU3 local0.err ReGaHss: ERROR: XMLRPC 'setValue' call failed (interface: 1007, params: {"KEQ1070840:2","STATE",true}) [CallSetValue():iseXmlRpc.cpp:1505]
Nov 18 10:43:29 CCU3 local0.err ReGaHss: ERROR: rpc.CallSetValue failed; address = KEQ1070840:2 [WriteValue():iseDOMdpHSS.cpp:76]
Nov 18 10:43:29 CCU3 local0.warn ReGaHss: WARNING: XMLRPC 'getValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"00089A498D06E6:5","ENERGY_COUNTER"}, result: [faultCode:-5,faultString:"Unknown Parameter value for value key: ENERGY_COUNTER"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2608]
Nov 18 10:43:29 CCU3 local0.err ReGaHss: ERROR: XMLRPC 'getValue' call failed (interface: 1009, params: {"00089A498D06E6:5","ENERGY_COUNTER"}) [CallGetValue():iseXmlRpc.cpp:1435]
Nov 18 10:43:29 CCU3 local0.err ReGaHss: ERROR: CallGetValue failed; sVal = 0.000000 [ReadValue():iseDOMdpHSS.cpp:124]
Nov 18 10:43:29 CCU3 local0.warn ReGaHss: WARNING: XMLRPC 'getValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"00189A498DE436:1","ALARMSTATE"}, result: [faultCode:-5,faultString:"Unknown Parameter value for value key: ALARMSTATE"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2608]
Nov 18 10:43:29 CCU3 local0.err ReGaHss: ERROR: XMLRPC 'getValue' call failed (interface: 1009, params: {"00189A498DE436:1","ALARMSTATE"}) [CallGetValue():iseXmlRpc.cpp:1435]
Nov 18 10:43:29 CCU3 local0.err ReGaHss: ERROR: CallGetValue failed; sVal = 0 [ReadValue():iseDOMdpHSS.cpp:124]
Nov 18 10:43:29 CCU3 local0.warn ReGaHss: WARNING: XMLRPC 'getValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"00021A4995E3E7:2","STATE"}, result: [faultCode:-5,faultString:"Unknown Parameter value for value key: STATE"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2608]
Nov 18 10:43:29 CCU3 local0.err ReGaHss: ERROR: XMLRPC 'getValue' call failed (interface: 1009, params: {"00021A4995E3E7:2","STATE"}) [CallGetValue():iseXmlRpc.cpp:1435]
Nov 18 10:43:29 CCU3 local0.err ReGaHss: ERROR: CallGetValue failed; sVal = 0 [ReadValue():iseDOMdpHSS.cpp:124]
Nov 18 10:43:29 CCU3 local0.warn ReGaHss: WARNING: XMLRPC 'getValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"00189A498DE436:0","LOW_BAT"}, result: [faultCode:-5,faultString:"Unknown Parameter value for value key: LOW_BAT"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2608]
Nov 18 10:43:29 CCU3 local0.err ReGaHss: ERROR: XMLRPC 'getValue' call failed (interface: 1009, params: {"00189A498DE436:0","LOW_BAT"}) [CallGetValue():iseXmlRpc.cpp:1435]
Nov 18 10:43:29 CCU3 local0.err ReGaHss: ERROR: CallGetValue failed; sVal = 0 [ReadValue():iseDOMdpHSS.cpp:124]
Nov 18 10:43:35 CCU3 daemon.info cuxd[895]: INIT 'xmlrpc_bin://127.0.0.1:31999' '1255'
Nov 18 10:43:36 CCU3 local0.warn ReGaHss: WARNING: XMLRPC 'getValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"00089A498D06E6:5","ENERGY_COUNTER_OVERFLOW"}, result: [faultCode:-5,faultString:"Unknown Parameter value for value key: ENERGY_COUNTER_OVERFLOW"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2608]
Nov 18 10:43:36 CCU3 local0.err ReGaHss: ERROR: XMLRPC 'getValue' call failed (interface: 1009, params: {"00089A498D06E6:5","ENERGY_COUNTER_OVERFLOW"}) [CallGetValue():iseXmlRpc.cpp:1435]
Nov 18 10:43:36 CCU3 local0.err ReGaHss: ERROR: CallGetValue failed; sVal = 0 [ReadValue():iseDOMdpHSS.cpp:124]
Nov 18 10:43:40 CCU3 user.info ReGaHss: SIGTERM: ReGaHss Halting
Nov 18 10:43:48 CCU3 user.warn monit[1284]: 'ReGaHss' failed protocol test [HTTP] at [localhost]:8183 [TCP/IP] -- Connection refused
Nov 18 10:44:05 CCU3 user.err monit[1284]: 'ReGaHss' service restarted 1 times within 1 cycles(s) - exec
Nov 18 10:44:05 CCU3 user.info monit[1284]: 'ReGaHss' exec: '/bin/triggerAlarm.tcl ReGaHss restarted WatchDog: regahss-restart true'
Nov 18 10:44:05 CCU3 user.info monit[1284]: 'ReGaHss' process is running after previous restart timeout (manually recovered?)
Nov 18 10:44:05 CCU3 local0.warn ReGaHss: WARNING: failed to add object id = 39 [SetState():iseDOMdp.cpp:258]
Nov 18 10:44:10 CCU3 local0.warn ReGaHss: WARNING: XMLRPC 'getValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"001A5A49985695:8","LEVEL"}, result: [faultCode:-5,faultString:"Unknown Parameter value for value key: LEVEL"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2608]
Nov 18 10:44:10 CCU3 local0.err ReGaHss: ERROR: XMLRPC 'getValue' call failed (interface: 1009, params: {"001A5A49985695:8","LEVEL"}) [CallGetValue():iseXmlRpc.cpp:1435]
Nov 18 10:44:10 CCU3 local0.err ReGaHss: ERROR: CallGetValue failed; sVal = 0.000000 [ReadValue():iseDOMdpHSS.cpp:124]
Nov 18 10:44:10 CCU3 local0.warn ReGaHss: WARNING: XMLRPC 'getValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"001A5A49985695:8","COLOR"}, result: [faultCode:-5,faultString:"Unknown Parameter value for value key: COLOR"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2608]
Nov 18 10:44:10 CCU3 local0.err ReGaHss: ERROR: XMLRPC 'getValue' call failed (interface: 1009, params: {"001A5A49985695:8","COLOR"}) [CallGetValue():iseXmlRpc.cpp:1435]
```

**Comment by jens-maus** [comment]
Sorry, but to me this looks like ReGaHss was busy doing stuff probably related to some WebUI programs and therefore could not serve the XMLRPC or other internal things. That caused monit to hard restart it to try to resolve the issue.

Please stop/disable all your WebUI programs, perform the update again and see if this solves the issue and then turn the WebUI programs back on. If this is the case make sure you don't have any loop in your WebUI programs or such.

**Comment by jp112sdl** [comment]
Corrupted `homematic.regadom`. Maybe a faulty sd card?
```
Nov 18 10:43:01 CCU3 local0.err ReGaHss: ERROR: Error compare close tag =/obj with endtag =dom [LoadObject():iseDOMpersist.cpp:515]
Nov 18 10:43:01 CCU3 local0.err ReGaHss: ERROR: failed load OM, file= /etc/config/homematic.regadom [LoadFromFile():iseDOMpersist.cpp:68]
Nov 18 10:43:01 CCU3 local0.err ReGaHss: ERROR: LoadFromFile failed. [LoadOM():iseDOM.cpp:2577]
```

**Comment by jens-maus** [comment]
Ah I missed that. Indeed! Then this looks like a user problem!

**Comment by tjschmid** [comment]
Sorry for this long brake. I wanted to provide a qualified feedback. But I had to organize something for analyzing (SD Card and Micro HDMI). At the end I was able to upgrade to the recent version (yesterday). 

I don’t know really the root cause as I changed at some point to much. I think there was something wrong with the internal diagrams and the additional USB stick on USB 2.0 port where the diagrams are stored. I recognized that the USB stick was really hot (more then normal). After unplugging USB / booting / deleting diagrams / re-plugging USB / rebooting my RaspberryMatic is working again.

Additional I checked the SD Card for faulty sectors or other issues but the tool found nothing. Thank you very much for this quick support and the help you provided.
